chpl/chpl-service/src/test/java/gov/healthit/chpl/validation/listing/reviewer/duplicate/AtlDuplicateReviewerTest.java

```diff
@@ -0,0 +1,121 @@
+package gov.healthit.chpl.validation.listing.reviewer.duplicate;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
+import gov.healthit.chpl.domain.CertifiedProductTestingLab;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+
+public class AtlDuplicateReviewerTest {
+    private static final String ERR_MSG =
+            "Listing contains duplicate Testing Lab: '%s'. The duplicates have been removed.";
+
+    private ErrorMessageUtil msgUtil;
+    private AtlDuplicateReviewer reviewer;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Not sure that this is necessary, since there are not Mockito annotation on any of the classes private members.